WcaOnRails/spec/features/sign_up_spec.rb

```diff
@@ -7,6 +7,13 @@
   let!(:person_without_dob) { FactoryBot.create :person, year: 0, month: 0, day: 0 }
   let!(:custom_delegate) { FactoryBot.create(:delegate) }
 
+  before :each do
+    # The cookie banner just gets in the way of these tests, and is already
+    # tested elsewhere. Set a cookie that prevents the cookie banner from
+    # appearing.
+    page.driver.set_cookie('cookie_eu_consented', 'true')
```

Comment:
Cool! We had been struggling to hide the cookie bar in testing before (the brute-force solution was to only render the cookie bar if `Rails.production?` returned `true`), would you mind sharing how you found out about this trick?

Reply:
That was tricky! My process went something like:

1. Google "rspec feature spec cookies". Give up after reading [this](https://stackoverflow.com/questions/29531499/how-to-test-cookies-with-rspec-and-capybara-in-a-feature-test) and [this](https://relishapp.com/rspec/rspec-rails/docs/controller-specs/cookies) (this is for controller specs, not feature specs, thanks google!)
2. Stop and think a bit. We use phantomjs, right?
3. Oh no [we don't anymore](https://github.com/thewca/worldcubeassociation.org/commit/b953fbb2e6f36ca4df0e22b9eec61df38528dfa8). Thanks @gregorbg!
4. Google "capybara apparition cookies".
5. Find https://github.com/twalpole/apparition#manipulating-cookies. Oooh this looks promising!

Reply:
Thanks, good to know! I also just realised that the name of the cookie is buried in [the gem's README](https://github.com/infinum/cookies_eu#cookies-used)